pkg/flowaggregator/flowaggregator.go

```diff
@@ -341,7 +342,7 @@ func (fa *flowAggregator) initExportingProcess() error {
 			CollectorProtocol:   fa.externalFlowCollectorProto,
 			ObservationDomainID: fa.observationDomainID,
 			TempRefTimeout:      1800,
-			PathMTU:             0,
+			PathMTU:             1400,
```

Comment:
this needs a comment...
it also seems unrelated to the rest of the PR

Reply:
Yes, it was not meant to be added. Removed it.